Form/Extension/ParsleyTypeExtension.php

```diff
@@ -141,6 +142,10 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         $constraints = $this->constraintBuilder->build();
 
         foreach ($constraints as $constraint) {
+            if (!$constraint instanceof NormalizableInterface) {
```

Comment:
Is there a reason your constraint doesn't extend `JBen87\ParsleyBundle\Validator\Constraint`?
If it did, you wouldn't need to perform that check.